kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -224,11 +224,18 @@ showClaim
     :: Claim claim
     => MonadState (ReplState claim) m
     => MonadWriter String m
-    => ClaimIndex
+    => Maybe ClaimIndex
     -> m ()
-showClaim cindex = do
-    claim <- getClaimByIndex . unClaimIndex $ cindex
-    maybe printNotFound (putStrLn' . showRewriteRule) $ claim
+showClaim mcindex =
```

Comment:
Since `mcindex` is only used for the case split, it makes more sense to use a `\case` here instead.